expertise/models/multifacet_recommender/specter.py

```diff
@@ -304,14 +329,34 @@ def load_emb_file(emb_file):
             print(len(bad_id_set))
             return emb_tensor, id_list, bad_id_set
 
-        with open(publications_path) as f_in:
-            print('Loading cached publications...')
-            paper_emb_train, train_id_list, train_bad_id_set = load_emb_file(f_in)
-            paper_num_train = len(train_id_list)
+        def load_from_redis():
+            paper_emb_size_default = 768
```

Comment:
what is `768` for?

Reply:
It's the default embedding size that BERT returns.